build/angular-slider.js

```diff
@@ -1395,7 +1410,10 @@ angular.module('vr.directives.slider', ['ngTouch']).directive('slider',
                                  */
                                 function redrawSliders() {
                                     setPointers();
-                                    adjustBubbles();
+                                    // Only adjust bubbles if allowed
+                                    if(scope.noBubbleAdjustment === undefined) {
```

Comment:
If we are going to submit this fork back, we should theoretically be following the style guide of the existing code.